configure.ac

```diff
@@ -406,6 +406,26 @@ case "${host_cpu}" in
 esac
 AC_DEFINE_UNQUOTED([CPU_SPINWAIT], [$CPU_SPINWAIT])
 
+AC_MSG_CHECKING([number of significant virtual address bits])
+case "${host_cpu}" in
+  x86_64|aarch64)
+    LG_VADDR=48
```

Comment:
According to that doc, the CPUID instruction exposes this info, so I'm inclined to make the configure test more robust.